CHANGELOG.md

```diff
@@ -3,6 +3,14 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.0.0] - 2021-06-13
+
+# Fix
+
+- Fixed percentages sometimes weren't correctly calculated, due to not taking border columns into account.
+- Return `None` for `Table::get_table_width`, if getting the terminal size somehow failed.
+- Fixed a lot of possible, but highly unlikely number conversion overflow issues.
+
```

Comment:
In my PR #36 I forgot to add to the change log, would you want to add the addition of the lines method in this PR?